pyproject.toml

```diff
@@ -1,3 +1,155 @@
+[build-system]
+requires = ["setuptools >= 61.1.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "optuna"
+description = "A hyperparameter optimization framework"
```

Comment:
Could you also set `long_description` attribute? I guess we can set it via `readme = "README.md"`.

Reply:
Thank you, added `readme` in e57d138. 🙏 
https://peps.python.org/pep-0621/#readme